spec/integ/pm.spec.js

```diff
@@ -294,4 +294,144 @@ describe("IRC-to-Matrix PMing", function() {
         yield createRoomPromise;
         yield sentMessagePromise;
     }));
+
+    it("should create a federated (by default) 1:1 matrix room and invite the real matrix user when " +
+    "it receives a PM directed at a virtual user from a real IRC user",
+    test.coroutine(function*() {
+        // mock create room impl
+        let createRoomPromise = new Promise(function(resolve, reject) {
+            sdk.createRoom.andCallFake(function(opts) {
+                expect(opts.visibility).toEqual("private");
+                expect(opts.invite).toEqual([tRealUserId]);
+                expect(opts.creation_content["m.federate"]).toEqual(true);
```

Comment:
This is the only line which is different to the earlier test; I would just do this assertion in that test and then we can remove this test entirely :D